contrib/passwd.bzl

```diff
@@ -54,17 +41,28 @@ passwd_file = rule(
         "info": attr.string(default = "user"),
         "home": attr.string(default = "/home"),
         "shell": attr.string(default = "/bin/bash"),
-        "build_tar": attr.label(
-            default = Label("@bazel_tools//tools/build_defs/pkg:build_tar"),
-            cfg = "host",
-            executable = True,
-            allow_files = True,
-        ),
     },
     executable = False,
     outputs = {
-        "out": "%{name}.passwd",
-        "tar": "%{name}.passwd.tar",
+        "out": "%{name}",
     },
     implementation = _impl,
 )
+
+def passwd_tar(name, username, uid, gid, info, home, shell, passwd_file_name="/etc/passwd"):
```

Comment:
At this point, I think it's cleaner to just have the user generate the tarball vs. understanding how `passwd_file_name` pollutes the label space with `basename(passwd_file_name)`.

Reply:
umm ok